src/test/resources/solidity/end_to_end_testing_big/swc/transaction_order_dependence/ERC20.json

```diff
@@ -64,12 +64,12 @@
           106,
           118
         ],
-        "violations": [],
-        "warnings": [
+        "violations": [
```

Comment:
Need to check this.

Reply:
The assembly code is just slightly reorganized between the two versions so this shouldn't result in a difference in pattern matches.

Reply:
Interesting. This is still with the regular decompiler included right?
Could it be that in one of the two cases (either old or new bytecode) the regular decompiler fails and therefore there is a difference?

Reply:
Yes in the old case the function decompiler succeeds.